### PR TITLE
Facedetection application files

### PR DESCRIPTION
**Pull request by oguzhanbsolak**

### CODE REVIEW

File: train_all_models.sh [hunk 1]
```diff
@@ -47,3 +47,6 @@ scripts/train_cifar100_qat8_effnet2.sh "$@"
 echo "-----------------------------"
 echo "Training Tiny SSD model with SVHN"
 scripts/train_svhn_tinierssd.sh "$@"
+echo "-----------------------------"
+echo "Training Tiny SSD model with FaceDet"
```
Comment (MaximGorkem) [inline train_all_models.sh:51]: "with FaceDet" gives the wrong impression that the FaceDet is the name of the dataset. We need something like "Training Tiny SSD face detection model"..

**Review by MaximGorkem — COMMENTED** [review]
Looks good

File: train_all_models.sh (hunk 1 above)
Comment (oguzhanbsolak) [reply train_all_models.sh]: I agree, changed the statement to "Training Tiny SSD face detection model" with the new commit.

File: models/ai85net-tinierssd-face.py [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+        self.fire6 = ai8x.FusedConv2dBNReLU(64, 64, 3, padding=1, **kwargs)
+        self.fire7 = ai8x.FusedConv2dBNReLU(64, 128, 3, padding=1, **kwargs)
+        self.fire8 = ai8x.FusedConv2dBNReLU(128, 32, 3, padding=1, **kwargs)
+
+        self.fire9 = ai8x.FusedMaxPoolConv2dBNReLU(32, 32, 3, padding=1,
+                                                   **kwargs)
+
+        self.fire10 = ai8x.FusedMaxPoolConv2dBNReLU(32, 32, 3, padding=1, **kwargs)
+
+    def forward(self, image):
```
Comment (ermanok) [inline models/ai85net-tinierssd-face.py:52]: Could you please arrange the content of the forward method of TinySSDBaseFace class. There too many blank lines.

**Review by ermanok — APPROVED** [review]
Looks good. I just have a single comment for code refactor.

File: models/ai85net-tinierssd-face.py (hunk 2 above)
Comment (oguzhanbsolak) [reply models/ai85net-tinierssd-face.py:52]: Realized in the new commits.

**Review by ermanok — APPROVED** [review]
Looks good.

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**